docs/guides/theme_architecture/blocks/schema/variables/variable_types/radio.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: default
+title: Radio
+parent: Variable types
+---
+
+# Radio
+
+Renders a radio button group and returns the value of the selected option.
+Takes an `options` parameter which defines the options in the radio button group, this is an array of label, value pairs.
+
+##### syntax
+{% raw %}
+```
+---
+my_variable:
+    type: radio
+    default: opt_1
```

Comment:
I assume if no default is mentioned the first option is default?

Reply:
Good point, will add this in